src/Adapter/Product/CommandHandler/UpdateProductSeoHandler.php

```diff
@@ -50,16 +51,24 @@ class UpdateProductSeoHandler implements UpdateProductSeoHandlerInterface
      */
     private $productSeoPropertiesFiller;
 
+    /**
+     * @var Tools
+     */
+    private $tools;
+
     /**
      * @param ProductMultiShopRepository $productRepository
      * @param ProductSeoPropertiesFiller $productSeoPropertiesFiller
+     * @param Tools $tools
      */
     public function __construct(
         ProductMultiShopRepository $productRepository,
-        ProductSeoPropertiesFiller $productSeoPropertiesFiller
+        ProductSeoPropertiesFiller $productSeoPropertiesFiller,
+        Tools $tools
```

Comment:
Please define the BC Break as the class exists on `1.7.8.x` and is not defined as `@internal`.